msteams-platform/tabs/how-to/authentication/tab-sso-register-aad.md

```diff
@@ -0,0 +1,345 @@
+---
+title: Register your tab app with Azure AD
+description: Describes registering your tab app with Azure AD
+ms.topic: how-to
+ms.localizationpriority: medium
+keywords: teams authentication tabs Microsoft Azure Active Directory (Azure AD) access token SSO tenancy scope 
+---
+# Register your app in Azure AD
+
+Azure AD provides access to your tab app based on the app user's Teams identity. You'll need to register your tab app with Azure AD so that the app user who has signed into Teams can be given access to your tab app.
+
+## Enabling SSO on Azure AD
+
+Registering your tab app in Azure AD and enabling it for SSO requires making app configurations, such as generating app ID, defining API scope, and configuring the On-behalf-of (OBO) flow.
+
+:::image type="content" source="../../../assets/images/authentication/teams-sso-tabs/register-azure-ad.png" alt-text="Configure Azure AD to send access token to Teams Client app" border="false":::
+
+Create a new app registration in Azure AD, and expose its (web) API using scopes (permissions). Configure a trust relationship between the exposed API on Azure AD and your app. The OBO flow allows app users to access your tab app without any further need of consent. You can add client IDs for the trusted mobile, desktop, and web applications that you want to pre-authorize.
```

Comment:
Suggest rewording to clarify that this is different from the application using OBO Flow which they will encounter when calling Graph. Something like:

> Create a new app registration in Azure AD and expose its (web) API using scopes (permissions). Configure a trust relationship between the exposed API on Azure AD and your app. This allows the Microsoft Teams client to use the OBO flow to obtain an access token on behalf of your application and the logged-in user. You can add client IDs for the trusted mobile, desktop, and web applications that you want to pre-authorize.